markdown_code_symlinks/__init__.py

```diff
@@ -106,20 +108,31 @@ def relative_docs(cls, url):
 
     @classmethod
     def add_mapping(cls, docs_rel, code_rel):
-        assert docs_rel not in cls.mapping['docs2code'], """\
-Assertion error! Document already in mapping!
+        assert_msg_temp = """\
+Document already in mapping!
     New Value: {}
 Current Value: {}
-""".format(docs_rel, cls.mapping['docs2code'][docs_rel])
+"""
+        sphinx_v3_0 = packaging.version.parse("3.0.0")
+        sphinx_v = packaging.version.parse(sphinx.__version__)
 
-        assert code_rel not in cls.mapping['code2docs'], """\
-Assertion error! Document already in mapping!
-    New Value: {}
-Current Value: {}
-""".format(docs_rel, cls.mapping['code2docs'][code_rel])
+        if sphinx_v < sphinx_v3_0:
+            if docs_rel in cls.mapping['docs2code']:
+                assert False, assert_msg_temp.format(docs_rel, cls.mapping['docs2code'][docs_rel])
```

Comment:
I would remove the if statement and leave the assert

```suggestion
            assert code_rel not in cls.mapping['code2docs'], assert_msg_temp.format(docs_rel, cls.mapping['docs2code'][docs_rel])
```

Reply:
Fixed